.circleci/config.yml

```diff
@@ -47,40 +54,22 @@ jobs:
           command: citus_indent --check
       - run:
           name: 'Fix whitespace'
-          command: ci/editorconfig.sh
-      - run:
-          name: 'Check if whitespace fixing changed anything, install editorconfig if it did'
-          command: git diff --exit-code
+          command: ci/editorconfig.sh && git diff --cached --exit-code
```

Comment:
I'm wondering should we really use `--cached` in those commands

Reply:
No we should not. Thanks for the catch.